docs/querying/sql-functions.md

```diff
@@ -1077,11 +1077,42 @@ Calculates the base-10 of the numeric expression.
 
 ## LOOKUP
 
-`LOOKUP(<CHARACTER>, <CHARACTER>[, <CHARACTER>])`
+Searches for `expr` in a registered [query-time lookup table](lookups.md) named `lookupName`. Returns `replaceMissingValueWith`, which defaults to `null` if `expr` is null or if the lookup doesn't contain a key for `expr`.
```

Comment:
Let's move the conditional to the start of the sentence so it doesn't look like the expected return behavior.
```suggestion
Searches for `expr` in a registered [query-time lookup table](lookups.md) named `lookupName` and returns the mapped value. If `expr` is null or not contained in the lookup, returns `replaceMissingValueWith` if supplied, otherwise null.
```

Reply:
Might need to update the table description too? Double check this

Reply:
Table description is just introducing the table that is being used, not how it is being used, I do not think it is necessary to update it

Reply:
Did need to be updated, changes are now live